tests/taskpool_wait/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+include(ParsecCompilePTG)
```

Comment:
All these tests should be in the directory `tests/api/taskpool_wait/`

Reply:
done.